src/components/CharacterLimitMainText/CharacterLimit.types.ts

```diff
@@ -1,7 +1,7 @@
-import { ReactNode } from 'react';
+import { IconsType } from '~components/Icon/Icon.types';
 
 export type ModuleProps = {
   maxLength: number;
-  svg: ReactNode;
+  svg: IconsType | null | undefined;
```

Comment:
por que undefined também? 
```suggestion
  svg: IconsType | null ;
```

Reply:
é que la no composerEditor acho que o socialMedias poderia ser undefined, aí eu adicionei que svg pode ser undefined e null